docker-compose.test.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.7"
+
+services:
```

Comment:
should this not also need a comment? just like `docker-compose.dev.yml`. for sake of consistency.